lib/home_page.dart

```diff
@@ -127,6 +140,41 @@ class HomeState extends State<Home> with WidgetsBindingObserver {
       });
     });
 
+    commentBloc = Provider.of<NostrCommentBloc>(context, listen: false);
```

Comment:
I don't see the NostrCommentBloc in the codebase. Did you forget to push?

Reply:
It is a part of any time, you will find it here https://github.com/breez/anytime_podcast_player/pull/48